docgen-templates/page.hbs

```diff
@@ -1,9 +1,10 @@
 {{h}} {{items.0.natspec.title}}
-{{items.0.natspec.notice}}
+{{{items.0.natspec.notice}}}
```

Comment:
I don't think this should have three brackets. I think it should have 2

Reply:
Needed to avoid the escape of characters like `